engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
Would be good to not have the remote stuff as part of engine-core, then exclusions like this are not needed anymore. Since we have many projects and products depending on the core we should keep it as light weight as possible and this is a good example of why.